src/components/settings/DiagnosticLogsPanel.tsx

```
@@ -0,0 +1,671 @@
+    }
+  }, []);
+
+  // Export logs
+  const handleExport = useCallback(async () => {
+    setIsExporting(true);
+    try {
+      const levelOrder: Record<LogLevel, number> = { debug: 0, info: 1, warn: 2, error: 3 };
+      const minLevel = selectedLevels.size > 0
+        ? (Object.entries(levelOrder).find(([l]) => selectedLevels.has(l as LogLevel))?.[1] ?? 3)
+        : 3;
+      const minLevelName = (Object.entries(levelOrder).find(([_, v]) => v === minLevel)?.[0] as LogLevel) || 'info';
```

ESLint

[error] '_' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'frontendLogs' is never reassigned. Use 'const' instead.

(prefer-const)